api/src/interpreter/interpreter.service.ts

```diff
@@ -174,4 +176,89 @@ export class InterpreterService {
       WHERE address IS NOT NULL
     `);
   }
+
+  async exportToWorkbook(): Promise<ExcelJS.Workbook> {
+    // Create workbook
+    const workbook = new ExcelJS.Workbook();
+    // Read template
+    await workbook.xlsx.readFile(path.join(__dirname, '..', '..', '/assets/Template_AllLanguages.xlsx'));
+
+    const worksheet = workbook.getWorksheet('Export');
+    if (!worksheet) {
+      throw new InternalServerErrorException('Unable to load template file to export data');
+    }
+    const setCell = setCellHelper(worksheet);
+
+    const interpreters: InterpreterEntity[] = await this.interpreterRepository.find();
+
+    let i = 2;
```

Comment:
```suggestion
   let cellRowIndex = 2
```
could we use `let cellRowIndex = 2`, make it easier to understand/semantic. what do you think?